betfund_solicitor/sendgrid.py

```diff
@@ -1,10 +1,21 @@
 """SendGrid V3 API wrapper class."""
 
+import json
+import os
+
+from betfund_logger import CloudLogger
 from python_http_client.exceptions import BadRequestsError
 from sendgrid import SendGridAPIClient
 
 from betfund_solicitor import Message
 
+logger = CloudLogger(
+    log_group="betfund-solicitor",
+    log_stream="sendgrid",
+    aws_access_key=os.environ.get("BETFUND_AWS_ACCESS_KEY", None),
```

Comment:
I believe the .get() by default returns `None` if value is not available via `os`

Reply:
Roger that!